Source/Scene/Expression.js

```diff
@@ -1298,6 +1318,11 @@ define([
                 if (value === 'TILES3D_TILESET_TIME') {
                     return 'u_tilesetTime';
                 }
+                break;
+            case ExpressionNodeType.TERNARY:
+                if (defined(ternaryFunctions[value])) {
+                    return value + '(' + left + ', ' + right + ', ' + test + ')';
+                }
```

Comment:
@lilleyse So, now the shader expression  test cases are passing, but the test cases for 'evaluates clamp function'  and 'evaluates mix function' don't make sense; for mix, "Expected NaN to equal 1." is the error message, and "DeveloperError: max is required." for clamp.  These don't make sense to me, since three arguments are passed for clamp and mix doesn't seem to be getting any arguments, given that the result is NaN.